config/routes.rb

```diff
@@ -25,6 +29,17 @@
   
   resources :payments, only: [:new, :create]
   
+  namespace :api do
+    controller :deals do
+      get 'deals/live' => :live
+      get 'deals/past' => :past
+    end
+
+    controller :users do
+      get 'my_orders' => :my_orders
```

Comment:
its mentioned in the docs, so that's why used it
![image](https://github.com/Ratchet1999/FlashSale/assets/62827625/8cbe7810-5e15-4c97-87a9-e27e05f51b5b)